src/Relation/Binary/Lattice/Properties/BoundedJoinSemilattice.agda

```diff
@@ -51,3 +54,22 @@ dualBoundedMeetSemilattice = record
   { ⊤                        = ⊥
   ; isBoundedMeetSemilattice = dualIsBoundedMeetSemilattice
   }
+
+-- Every bounded semilattice gives rise to a commutative pomonoid
+
+isPomonoid : CommutativePomonoid c ℓ₁ ℓ₂
```

Comment:
This should be called `commutativePomonoid`? Also the usual way to build these up is by defining the `is` structures one by one so that they are also available publicly in the file.

Reply:
Yep, sorry, will fix.

Reply:
Fixed.